src/test/scala/org/coroutines/regression-tests.scala

```diff
@@ -21,13 +21,11 @@ class RegressionTest extends FunSuite with Matchers {
     assert(c1.value == 5)
     assert(!c1.resume)
     assert(c1.isCompleted)
-    assert(c1.result == (()))
     val c2 = call(xOrY(-2, 7))
     assert(c2.resume)
     assert(c2.value == 7)
     assert(!c2.resume)
     assert(c2.isCompleted)
-    assert(c2.result == (()))
```

Comment:
Perfect, thanks!

But we would need to do the same in other places where you removed the call to `result`.